catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/SaxEventToXmlElementConverter.java

```diff
@@ -58,6 +61,11 @@
      */
     private Set<String> namespacesAdded = new HashSet<>();
 
+    /*
+     * Stack of scoped namespace URI to prefix mappings
+     */
+    private Deque<Map<String, String>> scopeOfNamespacesAdded = new ArrayDeque<>();
```

Comment:
Ok, and I'll used a synchronized collection and return a copy on the static guy, that should help protect it.